website/content/docs/operations/nomad-agent.mdx

```diff
@@ -38,6 +38,10 @@ The agent is started with the [`nomad agent` command](/nomad/docs/commands/agent
 command blocks, running forever or until told to quit. The agent command takes a variety
 of configuration options, but most have sane defaults.
 
+-> **Note:** If you are running Nomad on Linux, you'll need to run client agents
+as root (or with `sudo`) so that cpuset accounting and network namespaces work
+correctly.
+
 When running `nomad agent`, you should see output similar to this:
 
 ```shell-session
```

Comment:
Argh, one line away from being able to leave this as a suggestion.

I think we should add `sudo` to the example command below.